cxx_mkspecs/gxx_mkspecs.py

```diff
@@ -444,3 +446,35 @@ def cxx_openwrt_gxx53_mips(conf):
     # Note: libstdc++ might not be available on the target platform
     # Statically link with the C++ standard library
     conf.env['LINKFLAGS'] += ['-static-libstdc++']
+
+
+@conf
+def cxx_poky_gxx63_arm7(conf):
```

Comment:
```suggestion
def cxx_poky_gxx63_armv7(conf):
```
We use the armv7 postfix in the other mkspecs, so let's make this consistent.